pallets/network-membership/src/types.rs

```diff
@@ -0,0 +1,44 @@
+// This file is part of CORD – https://cord.network
+
+// Copyright (C) Dhiway Networks Pvt. Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// CORD is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// CORD is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with CORD. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::*;
+use codec::{Decode, Encode};
+use frame_support::dispatch::DispatchClass;
+use scale_info::TypeInfo;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
```

Comment:
Question: When the runtime gets compiled to WASM, do we have access to std? WASM is no_std right? If so, then how the serialisation and deserialisation will work?  Here we have `serde` as a feature dependency.

Reply:
We are using serde in a no_std context with derive feature for Serialize, Deserialize trait. check the cargo file